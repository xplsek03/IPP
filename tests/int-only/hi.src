<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode19">
 <instruction order="1" opcode="JUMP">
  <arg1 type="label">$$main</arg1>
 </instruction>
 <instruction order="2" opcode="LABEL">
  <arg1 type="label">$foo</arg1>
 </instruction>
 <instruction order="3" opcode="PUSHFRAME"/>
 <instruction order="4" opcode="DEFVAR">
  <arg1 type="var">LF@%retval</arg1>
 </instruction>
 <instruction order="5" opcode="MOVE">
  <arg1 type="var">LF@%retval</arg1>
  <arg2 type="nil">nil</arg2>
 </instruction>
 <instruction order="6" opcode="DEFVAR">
  <arg1 type="var">LF@param1</arg1>
 </instruction>
 <instruction order="7" opcode="MOVE">
  <arg1 type="var">LF@param1</arg1>
  <arg2 type="var">LF@%1</arg2>
 </instruction>
 <instruction order="8" opcode="DEFVAR">
  <arg1 type="var">LF@param2</arg1>
 </instruction>
 <instruction order="9" opcode="MOVE">
  <arg1 type="var">LF@param2</arg1>
  <arg2 type="var">LF@%2</arg2>
 </instruction>
 <instruction order="10" opcode="WRITE">
  <arg1 type="var">LF@param2</arg1>
 </instruction>
 <instruction order="11" opcode="WRITE">
  <arg1 type="var">LF@param1</arg1>
 </instruction>
 <instruction order="12" opcode="MOVE">
  <arg1 type="var">LF@%retval</arg1>
  <arg2 type="int">10</arg2>
 </instruction>
 <instruction order="13" opcode="POPFRAME"/>
 <instruction order="14" opcode="RETURN"/>
 <instruction order="15" opcode="LABEL">
  <arg1 type="label">$$main</arg1>
 </instruction>
 <instruction order="16" opcode="CREATEFRAME"/>
 <instruction order="17" opcode="PUSHFRAME"/>
 <instruction order="18" opcode="DEFVAR">
  <arg1 type="var">LF@y</arg1>
 </instruction>
 <instruction order="19" opcode="CREATEFRAME"/>
 <instruction order="20" opcode="DEFVAR">
  <arg1 type="var">TF@%1</arg1>
 </instruction>
 <instruction order="21" opcode="MOVE">
  <arg1 type="var">TF@%1</arg1>
  <arg2 type="int">10</arg2>
 </instruction>
 <instruction order="22" opcode="DEFVAR">
  <arg1 type="var">TF@%2</arg1>
 </instruction>
 <instruction order="23" opcode="MOVE">
  <arg1 type="var">TF@%2</arg1>
  <arg2 type="string">Hi\032X!</arg2>
 </instruction>
 <instruction order="24" opcode="CALL">
  <arg1 type="label">$foo</arg1>
 </instruction>
 <instruction order="25" opcode="MOVE">
  <arg1 type="var">LF@y</arg1>
  <arg2 type="var">TF@%retval</arg2>
 </instruction>
 <instruction order="26" opcode="WRITE">
  <arg1 type="var">LF@y</arg1>
 </instruction>
</program>
