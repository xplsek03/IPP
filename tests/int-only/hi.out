Hi X!1010